src/foam/u2/CheckBox.js

```diff
@@ -75,7 +75,7 @@ foam.CLASS({
 
   css: `
     ^ {
-      margin: 4px;
+      margin: 4px 0px 4px 0px;
```

Comment:
Is this this same as just margin: 4px 0;  ?

Reply:
yes